.github/workflows/ts_serialize_release.yml

```diff
@@ -52,13 +52,15 @@ jobs:
           release_name: Release ${{ github.ref }}
           draft: false
           prerelease: false
-      - name: Upload Release Asset
-        id: upload-release-asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: setup-node
+        uses: actions/setup-node@v1
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
-          asset_path: ./dist/ts_serialize.tar.gz
-          asset_name: ts_serialize.tar.gz
-          asset_content_type: application/gzip
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: deploy project
+        run: |
+          cd dist
+          npm install
```

Comment:
Is this `install` call here for generating the package lock file? If it is then we may want to move this into the "build" phase IMO.